### PR TITLE
kubie 0.10.0

### PR DESCRIPTION
**Pull request by c10l**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/kubie.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 class Kubie < Formula
   desc "Much more powerful alternative to kubectx and kubens"
   homepage "https://blog.sbstp.ca/introducing-kubie/"
-  url "https://github.com/sbstp/kubie/archive/v0.9.1.tar.gz"
-  sha256 "8f7926b1bcd0421247231353b6969c36498add3b88b55103349df3da763a5f42"
+  url "https://github.com/sbstp/kubie/archive/v0.10.0.tar.gz"
+  sha256 "67f09d8a2101192452c6c37a2e5228d62802618db3df2892afd699feb72b4877"
   license "Zlib"
+  revision 1
```
Comment (issyl0) [inline Formula/kubie.rb:7]:
```suggestion
```

**Review by issyl0 — CHANGES_REQUESTED** [review]
Thanks! One minor nit: this new version doesn't need a `revision` line as `revision` is to force upgrades - for example if we fixed a security issue in an existing formula version.

File: Formula/kubie.rb (hunk 1 above)
Comment (c10l) [reply Formula/kubie.rb]: Thanks, I wasn't sure it was needed. I've pushed again without the `revision` line.

**Review by issyl0 — APPROVED** [review]
Thanks for your contribution to Homebrew! 💪

**Pull request closed without merging**